metricflow/test/integration/test_cases/itest_metrics_no_dim.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Can we rename this test file to either `itest_metric_queries_no_dimensions` or similar?